cray_product_catalog/catalog_update.py

```diff
@@ -172,10 +184,16 @@ def update_config_map(data: dict, name, namespace):
             LOGGER.exception("Error calling read_namespaced_config_map")
 
             # ConfigMap doesn't exist yet
-            if err.status == ERR_NOT_FOUND:
+            if err.status != ERR_NOT_FOUND:
+                raise   # unrecoverable
+            if name == CONFIG_MAP:
+                # If main ConfigMap is not found wait until it is available
                 LOGGER.warning("ConfigMap %s/%s doesn't exist, attempting again", namespace, name)
-                continue
-            raise  # unrecoverable
+            else:
+                # If product ConfigMap is not available then create
+                LOGGER.warning("Product ConfigMap %s/%s doesn't exist, attempting to create", namespace, name)
+                create_config_map(api_instance, name, namespace)
```

Comment:
@lathanm , lets add a try catch here to handle cm creation issues. and get out gracefully.